helmfile/charts/notify-celery/values.yaml

```diff
@@ -16,7 +16,7 @@ celeryCommon:
   NOTIFY_ENVIRONMENT: "dev"
   NOTIFICATION_QUEUE_PREFIX: "eks-notification-canada-ca"
   REDIS_ENABLED: "1"
-  AWS_US_TOLL_FREE_NUMBER: "+18449521252"
+  AWS_US_TOLL_FREE_NUMBER: "+18449261787"
```

Comment:
The previous value is the staging toll free number and new one is production value. 👍